dotCMS/src/main/java/com/dotcms/util/ContentTypeUtil.java

```diff
@@ -107,7 +107,9 @@ public String getActionUrl(final ContentType contentType) {
 	 */
     public String getActionUrl(final ContentType contentType, final User user) {
       final HttpServletRequest request = httpServletRequestThreadLocal.getRequest();
-      return getActionUrl(request, contentType, user);
+      
+      String actionUrl = request != null? getActionUrl(request, contentType, user):null;
```

Comment:
you could change this to return request != null? getActionUrl(request, contentType, user):null;